drivers/ethernet/Kconfig.dsa

```diff
@@ -6,7 +6,7 @@
 
 menuconfig NET_DSA
 	bool "Distributed Switch Architecture support"
-	depends on ETH_MCUX || ETH_SAM_GMAC || ETH_STM32_HAL
+	depends on ETH_MCUX || ETH_SAM_GMAC || ETH_STM32_HAL || ETH_NXP_ENET
```

Comment:
Thanks for this addition, however this doesn't really scale, and doesn't allow for OOT drivers.

Maybe we need to add a hidden kconfig, something like `ETH_SUPPORT_DSA`, which is selected by those drivers?

```
config ETH_DSA_SUPPORT
	bool
	help
	  Hidden symbol to indicate DSA is supported by an Ethernet driver.

menuconfig NET_DSA
	bool "Distributed Switch Architecture support"
	depends on ETH_DSA_SUPPORT
```

Reply:
okay, I added a commit converting this